### PR TITLE
Heroku deploy github action

### PR DESCRIPTION
**Pull request by avan1235**

### CODE REVIEW

**Comment by kantoniak** [comment]
I just noticed `master` was merged into `server` here. `master` should never be merged into other branches. Please revert merge commit: `git revert 0b171f0 -m 1`

File: .github/workflows/webserver_deploy.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Webserver deploy
```
Comment (kantoniak) [inline .github/workflows/webserver_deploy.yml]: Cool!

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,9 @@
+.idea/
```
Comment (kantoniak) [inline .gitignore]: This file will have a conflict once `server` gets merged. We will have to wait for #11 to check in.

File: server/Pipfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
+[[source]]
+
```
Comment (kantoniak) [inline server/Pipfile]: Are these empty lines necessary? Or is it me not used to Python conventions? :)

File: server/home/admin.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```
Comment (kantoniak) [inline server/home/admin.py]: We're waiting for #11 to merge so these changes go away.

File: server/Pipfile (hunk 3 above)
Comment (avan1235) [reply server/Pipfile]: I think that as long as the code is consistent then the formatting is not necessary, not only in Python case.

**Pull request closed without merging**